server/v2/appmanager/appmanager.go

```diff
@@ -91,7 +91,24 @@ func (a AppManager[T]) InitGenesis(
 
 // ExportGenesis exports the genesis state of the application.
 func (a AppManager[T]) ExportGenesis(ctx context.Context, version uint64) ([]byte, error) {
```

Comment:
Shouldn't it used `version`? Here it gets the latest only right but `ExportGenesis` takes a version